src/modules/commentsPopUp.js

```diff
@@ -0,0 +1,123 @@
+import updateTotalNumberOfComments, {
+  fetchAllMovieComments, commentApi, movieApi,
+} from './commentsItemDetail.js';
+
+const commentPopUp = document.querySelector('.movie-popup');
+
+const getTvInfo = (tvUrl) => fetch(tvUrl)
+  .then((response) => response.json())
+  .then((actualData) => actualData)
+  .catch((error) => error);
+
```

Comment:
- [ ] It would be nice to use the `async` and `await` keywords with the `fetch` function to get data from the API endpoint according to project requirements. Kindly implement this change. 👍